superdesk/wsgi.py

```diff
@@ -0,0 +1,11 @@
+"""WSGI entrypoint.
+
+Usage via Procfile:
+
+    rest: gunicorn -c pythong:superdesk.gunicorn_config superdesk.wsgi
+
+"""
+
+from superdesk.factory import get_app
+
+application = get_app()
```

Comment:
by the way, maybe `superdesk.worker` should be renamed to `superdesk.celery` then for more explicit :)